opentelemetry-sdk/src/opentelemetry/sdk/util.py

```diff
@@ -26,6 +28,31 @@
     from collections import Sequence
 
 
+def set_timeout_signal_handler():
+    "Signal timeout setter."
+
+    def signal_handler_function(signum, frame):
+        raise TimeoutError
+
+    return signal.signal(signal.SIGALRM, signal_handler_function)
+
+
+@contextmanager
+def timeout_in_seconds(seconds=None):
```

Comment:
I think is better if you document the function attribute instead of using the name of the function as documentation for the attribute itself, something like this:

```python
def timeout(time):
    """
    ...
    time: time to wait before timing out, in seconds
    """
    ...
```

Reply:
@ocelotl you and @toumorokoshi can duke it out over [hungarian notation](https://github.com/open-telemetry/opentelemetry-python/pull/186#discussion_r329295584). 😄